pkg/github/github_helper_debug_test.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package github
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+// THIS FILE IS NOT UNIT TESTS
```

Comment:
I don't think it's the scope of this PR, but I was wondering if it's worth to create a test user/repository in Github so we could make these tests here work like integration tests. I think they cover the classes in this pkg pretty well.

Reply:
It's hard to say. Obviously, we cannot use a common repository, only a common template. Then, to run tests, we have to configure/reuse GitHub credentials, clone template repository, clean it up after success or fail. This is not straightforward flow for in-repo tests. I'd leave it for more complex e2e test on CI. Moreover, it's unlikely that interactions with existing GitHub API would change. Even if it happens, it would be a rare case. Similar situation with changes in code: if someone touches parts that related to GitHub interactions it's easy to test and make sure they work as expected. However, I still don't have strong opinion on this. If you think we need such tests - please create an issue.

Reply:
Yeah, it's definitely a more-complicated-than-usual type of test. I was mostly trying to figure out how to have this part covered with tests, but I do agree that the e2e tests should be a better place to have this functionality covered. So let's leave this as is.